stack.yaml

```diff
@@ -1,66 +1,15 @@
-# This file was automatically generated by 'stack init'
-#
-# Some commonly used options have been documented as comments in this file.
-# For advanced use and comprehensive documentation of the format, please see:
-# http://docs.haskellstack.org/en/stable/yaml_configuration/
+resolver: lts-7.16
+compiler: ghc-8.0.2
 
-# Resolver to choose a 'specific' stackage snapshot or a compiler version.
-# A snapshot resolver dictates the compiler version and the set of packages
-# to be used for project dependencies. For example:
-#
-# resolver: lts-3.5
-# resolver: nightly-2015-09-21
-# resolver: ghc-7.10.2
-# resolver: ghcjs-0.1.0_ghc-7.10.2
-# resolver:
-#  name: custom-snapshot
-#  location: "./custom-snapshot.yaml"
-resolver: lts-8.0
```

Comment:
Yes, I simply matched the resolver in `cardano-crypto` with one in `cardano-sl`. At first I did this while trying to resolve my local building issues, the issues though were due to lack of `nix` section specified, but I left it as is. I think it's better to have similar lts in `cardano-sl` and `cardano-crypto` though, to avoid possible problems while building `cardano-crypto` as a dependency.
Don't remember why we stick with 7.16 in `cardano-sl` though, but we probably have reasons (apart from avoiding download of new lts)